### ISSUE TITLE
Running netopeer2-cli as a single command

### ISSUE THREAD

**Issue by esend7881** [CESNET/netopeer2#783]
I'd like to run an `edit-config` using `netopeer2-cli` but as a single command.

I.e., currently one must:
```.sh
$ netopeer2-cli
> connect
> edit-config --target=running --file=..
```

But is there a mode where I can run netopeer2-cli all in one line in my shell? I.e., 
```.sh
$ netopeer2-cli --run-command 'edit-config --target=running --file=...' --user root --pass root
```

When I look for the `netopeer2-cli --help` nothing comes up in that form.

#### Env:
- libyang1-1.0.184-7.2.x86_64.rpm
- sysrepo-1.4.70-2.3.x86_64.rpm
- libnetconf2-1.1.26-2.3.x86_64.rpm
- netopeer2-1.1.39-2.3.x86_64.rpm

**Comment by michalvasko** [comment]
Look into `netopeer2/cli/sample_script.sh`. It is possible to write a script that will perform whatever commands you want.

**Comment by esend7881** [comment]
> Look into `netopeer2/cli/sample_script.sh`. It is possible to write a script that will perform whatever commands you want.

This looks promising, however I'll need to programmatically get around the interactivity of the `connect` command. Looks like my options are:
```
connect [--help] [--ssh] [--host <hostname>] [--port <num>] [--login <username>]
connect [--help] --tls [--host <hostname>] [--port <num>] [--cert <cert_path> [--key <key_path>]] [--trusted <trusted_CA_store.pem>]
connect [--help] --unix [--socket <path>]
```

**Comment by robbynet** [comment]
Hello,
To remove interactivity at connect level, key authentication is a good solution.
Also, in such study, Python module ncclient needs to be assessed as well.
Regards.
Christian.

**Comment by michalvasko** [comment]
Yeah, what @robbynet says, SSH keys avoid all interactivity. Also, *netopeer2-cli* is mostly an example client that should work well but with only basic functionality. So if you require more, you should probably look for some other client.

**Comment by esend7881** [comment]
@michalvasko / @robbynet - is there documentation on how to generate keys for netopeer2-server/client? 

I'm not sure if I can just use `ssh-keygen` and copy the id_rsa files to certain locations or not, thought I would ask you first.

**Comment by robbynet** [comment]
From my simple End User view, via "auth" cmd, CLI is flexible enough to avoid any constraint on keys installation. In my env to avoid having to enter pwd, CLI has been configured as follow : 
> auth keys
The keys used for SSH authentication:
#0: /home/netconf/.ssh/id_dsa.pub (private /home/netconf/.ssh/id_dsa)
> auth pref
The SSH authentication method preferences:
        'publickey':   3
        'password':    1
        'interactive': 2
>

**Comment by michalvasko** [comment]
@esend7881 You can generate the keys using `ssh-keygen` but you must set them using CLI command `auth`. No need to copy them anywhere, this command takes care of that.

As for RPMs, no, we do not provide separate packages of individual *netopeer2* binaries.
